src/components/Layout/MenuDrawer/MenuDrawer.tsx

```diff
@@ -18,6 +24,11 @@ interface MenuDrawerProps {
 }
 
 const MenuDrawer: React.FC<MenuDrawerProps> = ({ openMenu, setOpenMenu }) => {
+  const [isLoginModalVisible, setIsLoginModalVisible] = useState(false);
```

Comment:
We don't need it use openMenu, setOpenMenu

Reply:
[Screencast_20240114_003851.webm](https://github.com/sef-global/scholarx-frontend/assets/48531182/4d317c0d-2b4f-4876-806a-3ac133e20679)

Fixed that issue :)